src/common/informixHelper.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Contains Informix helper methods for loading reviews from Informix 
+ */
+const config = require('config')
+const logger = require('./logger')
+const helper = require('./helper')
+const informix = require('informixdb')
+const ReviewService = require('../services/ReviewService')
+const ReviewSummationService = require('../services/ReviewSummationService')
+/*
+ * This function loads the online review details for a given submission from Informix. 
+ * It uses the data to create review and reviewSummation objects which are then saved
+ * back to DynamoDB through the relevant services, and to ES through Bus API messages
+ * processed by the submission-processor-es code
+ */
+async function loadOnlineReviewDetails (authUser, submission) {
+  const reviewSummation = {}
+  const reviewsCreated = []
+  // We can only load in OR details from the legacy submission ID.
```

Comment:
Can we add some spaces between the major blocks of code here, just to help my old, tired eyes more easily scan it?